common/constants/navigation.js

```diff
@@ -21,28 +21,29 @@ const eventsLink = {
   href: '/events',
 };
 
+export const loginLinks = {
```

Comment:
Ah now I see.

Why not simply make two arrays of navItems? One for loggedIn and one for loggedOut. Export/Import both and choose the right one based off the state variable at the implementation level.